drivers/bluetooth/hci/Kconfig

```diff
@@ -234,3 +234,45 @@ config BT_SILABS_HCI_BUFFER_MEMORY
 	help
 	  Select the size of allocated memory buffer for the Silicon Labs
 	  Bluetooth Library.
+
+config BT_NXP_CTLR
+	bool "NXP Bluetooth Controller [EXPERIMENTAL]"
+	select GPIO
+	# select UART_USE_RUNTIME_CONFIGURE
+	select EXPERIMENTAL
+	depends on BT_H4
+	help
+	  Enables support for NXP Bluetooth Controller.
+	  Murata 2EL M.2 Moudle base on NXP IW612 Chipset which supports
+	  Wi-Fi® 802.11a/b/g/n/ac/ax + Bluetooth® 5.3 BR/EDR/LE + IEEE802.1.5.4
+	  up to 601 Mbps data rate on Wi-Fi® and 2Mbps data rate on Bluetooth®.
+	  4-wire UART@3M baud is supported. PCM for audio is also supported.
+	  Details of the module could be fond on
+	  https://www.embeddedartists.com/products/2el-m-2-module/
+
+	  The configuration depends on the firmware (named uart_nw61x_se.h)
+	  of Murata 2EL M.2 Moudle. It could be downloaded here
+	  https://github.com/NXP/wifi_nxp/blob/MCUX_2.15.000/wifi_bt_firmware/nw61x/uart_nw61x_se.h
+	  And it needs to be placed into the folder <zephyrproject>/zephyr/
+	  drivers/bluetooth/hci.
```

Comment:
This needs to be driving via binary blob support in the NXP HAL.

Reply:
Yes. Updated.

Reply:
@lylezhu2012, to update you will need to change the west.yml to point to the NXP HAL PR: https://github.com/zephyrproject-rtos/hal_nxp/pull/381

Reply:
Yes. It is updated here https://github.com/zephyrproject-rtos/zephyr/pull/70815/commits/a5c46c9e4bb2f3cee1c977b4878fabc5ab3067ad